logical/plugin/logger.go

```diff
@@ -202,3 +108,25 @@ type LoggerReply struct {
 	IsTrue bool
 	Error  error
 }
+
+func translateLevel(logxiLevel int) hclog.Level {
+
+	switch logxiLevel {
+
+	case logxi.LevelAll, logxi.LevelTrace:
```

Comment:
If we simply pull these into consts in the file we can get rid of the logxi dependency instead of keeping it around just for this. Also, what's the rationale here; do we have a way of knowing for sure whether or not the other end is using logxi levels? If not, I suggest we just assume the other end is using hclog levels. All of our plugins can be updated properly, and third party plugins can adapt. @briankassouf any guidance here?

Reply:
Yeah this is only here so that already compiled plugins will continue to work. Newly compiled plugins will no longer use a RPC transport for logs, since with this change gRPC and netRPC use go-plugin to forward logs.

Reply:
OK so this is a "when we drop support for netRPC (in 0.11?) we can remove thing?

Regardless I think we could duplicate these constants instead of having to actually keep logxi around in deps.

Reply:
Correct!